gradio/blocks.py

```diff
@@ -1708,7 +1697,6 @@ def launch(
         inbrowser: bool = False,
         share: bool | None = None,
         debug: bool = False,
-        max_threads: int = 40,
```

Comment:
I think we should still keep this parameter. It's not the same as the concurrency_count of the queue. You may want to increase it to match the concurrency_count for example.

Reply:
I'll restore it in this PR, but do we need it really?  Seems like controlling the number of workers is the main thing that a user would care about cc @abidlabs

Reply:
I agree the concurrency count of the queue is the main parameter users will want to control. But the way I understand it, starlette will use the same threadpool to run all the functions (unless they're coroutines). That's what `anyio.to_thread.run_sync` is doing. So if we remove the param, only 40 functions can only run concurrently.

Maybe we can set max_threads to always match the concurrency_count but I still don't think that's the best solution. Since events can still hit the predict route directly.